composer.json

```
@@ -30,7 +30,7 @@
         "contao/core-bundle": "^4.13 || ^5.0",
         "contao/manager-plugin": "^2.3.1",
         "doctrine/dbal": "^3.7",
-        "hwi/oauth-bundle": "^2.1",
+        "hwi/oauth-bundle": ">=2.1 <2.2",
```

```suggestion
        "hwi/oauth-bundle": "2.1.*",
```
would be more readable, no?